lib/rubocop/cop/style/align_parameters.rb

```diff
@@ -27,8 +27,13 @@ def on_method_def(node, _method_name, args, _body)
 
         def message(node)
           type = node && node.parent.send_type? ? 'call' : 'definition'
-          "Align the parameters of a method #{type} if they span " \
-          'more than one line.'
+          if fixed_indentation?
+            'Use one level of indentation for parameters following the first ' \
+              "line of a multi-line method #{type}"
```

Comment:
This message should end with a `.`. I'd also move two strings into constants and use `format` on them.